taskcluster/docker/android-build/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# %ARG DOCKER_IMAGE_PARENT
```

Comment:
This is a much simplified version of https://searchfox.org/mozilla-central/rev/2cd2d511c0d94a34fb7fa3b746f54170ee759e35/taskcluster/docker/android-build/Dockerfile. I removed anything that seems related to building gecko. Only remained the installation of nexus itself. I tweaked it to make the final image smaller.